### ISSUE TITLE
Final board? 

### ISSUE THREAD

**Issue by dronecz** [MickMad/JACK#1]
Hi @MickMad 

are these files for your board final? I´m looking for some small board with SAMD21G18 and I like your design. I would like place an order to PCB fab so please let me know. 

Thanks

**Comment by MickMad** [comment]
Not at all, the pins are oddly mapped, there's no silkscreen for their label, the board is not as small as it could be, and since I started working on HACK (https://github.com/MickMad/HACK) I stopped improving on this design. I will take it up again when I tackle I2S on this chip. You may take HACK as a starting point, but bear in mind there could be some layout issue there too. :)

**Comment by dronecz** [comment]
Thanks for reply. I looked at HACK and to be honest I like more JACK and I donť use Kicad so HACK is not usable for now. Can you tell me what worked and what not on JACK? I mean things like loading sketches, SPI, I2C and things like that. If I make this board and populate it, will be working?

**Comment by MickMad** [comment]
The prototypes I have here worked, but there are some issues. USB won't work in sketch mode, which is a big drawback. I'm sure there's something going on with the clock generator that does not like the fact that we're not running the thing with a 32.768 KHz crystal. Plus, the code (for both modified bootloader and modified core) is not there anymore, you may have to rewrite it. I might check on the PC if I have some old version left over.  Also, there's a known issue with the reset-to-bootloader thingy (it's called double-tap-reset but it works once every 10 times), you may want to enable the bootloader pin.
